src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/authorization/LocalUserCreator.java

```diff
@@ -8,6 +8,9 @@
 import de.dhbw.karlsruhe.students.mailflow.core.domain.user.User;
 
 /**
+ * is allowed to use Address object because it is only called in a service and not in the external
+ * layer
+ *
```

Comment:
this shouldn't be a doc-comment if at all necessary.

A doc-comment should provide a high-level information for another API user and do not document technical implementation details.